.github/workflows/ci.yml

```
@@ -258,6 +261,67 @@ jobs:
           BABEL_ENV: test
           BABEL_8_BREAKING: true
           BABEL_TYPES_8_BREAKING: true
+      - uses: actions/upload-artifact@v3
+        with:
+          name: babel8-test-artifact
+          path: |
+            codemods/*/lib/**/*
+            eslint/*/lib/**/*
+            packages/*/lib/**/*
+            packages/babel-standalone/*.js
+            !**/node_modules/**
+
+  test-babel-8-breaking:
+    name: Test Babel 8 breaking changes
+    needs: build-babel8
+    runs-on: ubuntu-latest
```

If test steps of Windows / Ubuntu are same, we can make it a test matrix.
```suggestion
    strategy:
      matrix:
        os: [ubuntu-latest, windows-latest]
    runs-on: ${{ matrix.os }}
```